plugins/FeatureFlags/Commands/EnableFeatureFlag.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\FeatureFlags\Commands;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugins\FeatureFlags\Commands\FeatureFlagFinder\FeatureFlagFinder;
+use Piwik\Plugins\FeatureFlags\FeatureFlagStorageInterface;
+
+class EnableFeatureFlag extends ConsoleCommand
+{
+    protected function configure()
+    {
+        $this->setName('featureflags:enable');
+        $this->setDescription('Enable a given feature flag)');
```

Comment:
```suggestion
        $this->setDescription('Enable a given feature flag');
```

The extraneous bracket is also in the other two command descriptions.

The other two command descriptions are written in a different style ("Disables" instead of "Disable"). Have no strong opinion on which one to choose, but all three should read the same.

Reply:
Fixed !